x-pack/legacy/plugins/reporting/export_types/common/execute_job/decrypt_job_headers.test.ts

```diff
@@ -43,7 +43,7 @@ describe('headers', () => {
     };
 
     const encryptedHeaders = await encryptHeaders(headers);
-    const { decryptedHeaders } = await decryptJobHeaders({
+    const decryptedHeaders = await decryptJobHeaders({
```

Comment:
Pipeline methods no longer need to have return value destructured, because they (usually) only return what is necessary. They no longer need to return a caravan of objects to get carried down the pipeline.